lib/project_types/extension/features/runtimes/checkout_ui_extension.rb

```diff
@@ -0,0 +1,26 @@
+module Extension
+  module Features
+    module Runtimes
+      class CheckoutUiExtension < Default
+        IDENTIFIERS = [
+          "CHECKOUT_ARGO_EXTENSION",
+          "CHECKOUT_UI_EXTENSION",
+        ]
+
+        AVAILABLE_FLAGS = [
+          :port,
+          :public_url,
+          :shop,
```

Comment:
Should we add the `resource_url` as well? It will be supported after this PR ships: https://github.com/Shopify/ui-extensions/pull/161

Reply:
@vividviolet I was going to do that on a separate PR :)